### PR TITLE
update package description

### PR DESCRIPTION
**Pull request by bgruening**
Hi @davebx,

I was not able to use the new R-3.2 packages. I always get a libblas error stating that FORTRAN_1_4 symbol was not found. I do no get these error with your old packages R < 3.2. Also when I compare the both packages R3.1 vs R3.2 it seems that R3.1 contains a lot more shared-object files, including cairo, Rblas.so, Ratlas.so ...

If you remove `--with-blas & --with-lapack` you get the Ratlas.so file back and installation of DESeq2 works again for me. Also please see this mailing list thread: http://dev.list.galaxyproject.org/Installing-Cairo-into-Gakaxy-td4667769.html

While I'm not sure this new build will fix the cairo issue, probably not, it fixes at least the blas, lapack issue. And a rebuild is needed to get this package in: https://github.com/galaxyproject/tools-iuc/pull/20

Thanks,
Bjoern

### CODE REVIEW

File: R/build.sh [hunk 1]
```diff
@@ -49,7 +47,11 @@ apt-get -qq update &&
     sed -i 's#/build/dest#$(R_ROOT_DIR)#g' /build/dest/lib/R/etc/Makeconf &&
     cp /usr/lib/libgfortran* /build/dest/lib/ &&
     cp /usr/lib/libgomp* /build/dest/lib/ &&
-    cp /usr/lib/libblas.so.3gf /build/dest/lib &&
+    cp /usr/lib/libblas* /build/dest/lib -R &&
```
Comment (natefoo) [inline R/build.sh]: Is the `-R` intentional? If so, it should probably precede the file arguments (although I'm not surprised it works here).

**Comment by martenson** [comment]
@galaxybot test this

File: R/build.sh (hunk 1 above)
Comment (bgruening) [inline R/build.sh]: Yes, there is a folder called `/usr/lib/libblas`. It works like it is :)

**Comment by bgruening** [comment]
@natefoo changed it.
@davebx any comments on this? Would be nice to see your implementation of R.

**Pull request closed without merging**